rad/src/lib.rs

```diff
@@ -109,9 +123,19 @@ pub async fn run_retrieval_report(retrieve: &RADRetrieve) -> Result<RadonReport<
                     message: x.to_string(),
                 })?;
 
-            let result = run_retrieval_with_data_report(retrieve, response_string, context);
-
-            log::debug!("Result for URL {}: {:?}", retrieve.url, result);
+            let result =
+                run_retrieval_with_data_report(retrieve, response_string, context, settings);
+
+            match &result {
+                Ok(report) => {
+                    log::debug!(
+                        "Successful result for source {}: {:?}",
+                        retrieve.url,
+                        report
+                    );
+                }
+                Err(e) => log::debug!("Failed result for source {}: {:?}", retrieve.url, e),
```

Comment:
I'm unable to see this line, I think it only happens if the RADON script is malformed, but they are checked when sending requests... And since HTTP errors are handled above, they are not printed here. So currently an HTTP error is not logged and if we see an HTTP error we must guess the URL. But let's leave that for another issue, or this PR will never be merged :laughing:

Reply:
Created issue #1289